lib/utils/__tests__/validateOptions.test.js

```diff
@@ -294,8 +281,8 @@ describe("validateOptions with a function for 'possible'", () => {
       ],
     })
     expect(validArrayOfObjectsAndStrings).toBe(true)
-    expect(result.warn.called).toBeFalsy()
-    result.warn.reset()
+    expect(result.warn.mock.calls).toHaveLength(0)
```

Comment:
This is fine; but in all these places, you could have also used [`toHaveBeenCalledTimes`](https://facebook.github.io/jest/docs/expect.html#tohavebeencalledtimesnumber).